src/openMVG/localization/CCTagLocalizer.hpp

```diff
@@ -1,5 +1,3 @@
-#ifdef HAVE_CCTAG
```

Comment:
The file is no more installed if we don't use CCTag.
This avoid the dependency propagation.